disc/testing.go

```diff
@@ -129,7 +129,7 @@ func (m *mockClient) PutEntry(ctx context.Context, sk cipher.SecKey, e *Entry) e
 	}
 }
 
-// AvailableServers returns all the servers that the APIClient mock has
+// AvailableServers returns all available the servers that the APIClient mock has
```

Comment:
Change `available the servers` to `available servers`

Reply:
Done.